source/includes/stackpath/_scripts.md

```diff
@@ -201,7 +201,56 @@ async function handleRequest(request) {
 }
 ```
 
-There following attributes are returned as part of the response.
+The following attributes are returned as part of the response.
+
+Attributes | &nbsp;
+------- | -----------
+`taskId` <br/>*string* | The task id related to creation of the script.
+`taskStatus` <br/>*string* | The status of the operation.
+
+
+<!-------------------- EDIT A SCRIPT -------------------->
+
+### Edit a script
+
+Edit a script in a given [environment](#administration-environments) for a given site ID.
+
+```shell
+curl -X PUT \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/scripts?siteId=:siteId"
+```
+> Request body example for editing a script:
+
+```json
+{
+  "routes": ["route"],
+  "code": "console.log('hello')"
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "ef70cafa-0544-4709-a66a-c68595ee105a",
+  "taskStatus": "SUCCESS"
```

Comment:
Would the `taskStatus` be PENDING rather than SUCCESS if the call is async? The documentation for the create script has the JSON response as PENDING, so my intuition tells me this may be PENDING as well

Reply:
Thanks for pointing that out.  Changed it to `PENDING` https://github.com/cloudops/cloudmc-api-docs/pull/333/commits/8af9039392360789a2bbbd82eeeb29ab80428134